lib/takeaway_class.rb

```diff
@@ -0,0 +1,38 @@
+class Takeaway
+
+	attr_reader :menu, :order, :actualcost
+
+	def initialize
+		@menu = {"Cheeseburger" => 1,
+			"Pizza" => 2,
+			"Fries" => 3
+		}
+		@actualcost = 0
+	end
```

Comment:
You might want to separate responsibilities of your menu and order into different classes.